lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -1257,12 +1279,33 @@ TEE_Result crypto_mac_init(void *ctx, uint32_t algo, const uint8_t *key,
 #endif
 #if defined(CFG_CRYPTO_CMAC)
 	case TEE_ALG_AES_CMAC:
-		return TEE_ERROR_NOT_SUPPORTED;
+		cipher_info = get_cipher_info(TEE_ALG_AES_ECB_NOPAD, len * 8);
+		if (!cipher_info)
+			return TEE_ERROR_NOT_SUPPORTED;
+
+		lmd_res = mbedtls_cipher_setup_info(ctx, cipher_info);
```

Comment:
Please comment why it's safe to change cipher_info here.

Reply:
Ok.